Models/MuZero_torch_agent.py

```diff
@@ -62,24 +64,22 @@ def __init__(self):
         super().__init__()
         self.full_support_size = config.ENCODER_NUM_STEPS
 
-        self.representation_network = mlp(config.OBSERVATION_SIZE, [config.HEAD_HIDDEN_SIZE] *
-                                          config.N_HEAD_HIDDEN_LAYERS, config.LAYER_HIDDEN_SIZE)
+        self.representation_network = mlp(config.OBSERVATION_SIZE, [config.LAYER_HIDDEN_SIZE] *
+                                          config.N_HEAD_HIDDEN_LAYERS, config.HIDDEN_STATE_SIZE)
 
-        self.action_encodings = mlp(config.ACTION_CONCAT_SIZE, [config.HEAD_HIDDEN_SIZE] * config.N_HEAD_HIDDEN_LAYERS,
-                                    config.LAYER_HIDDEN_SIZE)
+        self.action_encodings = mlp(config.ACTION_CONCAT_SIZE, [config.LAYER_HIDDEN_SIZE] * 0,
```

Comment:
Why are there no hidden layers?

Reply:
To match https://github.com/google-research/google-research/blob/master/muzero/network.py#L93, action encoding is a single layer with no activations